edge/pkg/edgehub/process.go

```diff
@@ -17,11 +17,10 @@ import (
 	"github.com/kubeedge/kubeedge/edge/pkg/edgehub/config"
 )
 
-var groupMap = map[string]string{
-	"resource": modules.MetaGroup,
-	"twin":     modules.TwinGroup,
-	"func":     modules.MetaGroup,
```

Comment:
it seems that group `func` is never used, can be deleted directly? 
cc @wackxu

Reply:
yes, we can delete it since we do not have edge function